client/src/components/new-request/analyze-pending.vue

```diff
@@ -82,3 +82,8 @@ export default class AnalyzePending extends Vue {
 }
 
 </script>
+<style scoped lang="scss">
+#analyze-pending-container {
+  max-width: 1140px;
```

Comment:
I'm beginning to think we should centralize (ie, in layout.scss) all there viewport width specs.

Reply:
Sorry i already did that, and forgot to remove this!
Currently as sass, but that will be converted when we convert the base style files.